packages/utils/src/tests/period/period.test.js

```diff
@@ -108,6 +108,177 @@ context('periodTypes', () => {
     });
   });
 
+  describe('comparePeriods', () => {
```

Comment:
I know this is a pain, but why not add a test case which requires that it be a certain number rather than just >0? In future, code may rely on this functionality.

Reply:
If code relies on that, it is an implementation detail - the specs of the function are the ones defined in the JsDoc, and the ones we need to assert in the test. The specs were based on `compareFunction` which can be passed in [Array.prototype.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort). 

Overly specific assertions make for fragile tests

Reply:
What I didn't assert is this

> compareFunction(a, b) must always return the same value when given a specific pair of elements a and b as its two arguments. If inconsistent results are returned, then the sort order is undefined.

i.e don't care if `compareFunction(a, b)` returns 3 or 5, as long as it is greater than zero - but it must return the same value every time. Happy to add a test case for those!

Reply:
Actually, I changed my mind - of course the same output should be returned with the same input, the implementation is clearly deterministic so need to add a test case for that

Reply:
That makes sense! In my head `comparePeriods` was more like 'return the number of days between these two periods', but he docstring makes the true specs clear.

Reply:
What you say is correct in this specific implementation, but it just happened that I implemented it this way - a different implementation with other integer values could work as well

What we want to achieve here is a classic C-inherited way of allowing a three-state return value instead of a binary `true`/`false` - the fact that one can convert entities to numbers and then substract them has historically made developers to define those states as `> 0`, `= 0`, `< 0`

Reply:
That makes sense. I'm sure you're used to it, but it's kind of strange when you think about it the first time.